axis.py

```diff
@@ -364,6 +364,7 @@ class BibliographicParser(Axis360Parser):
     DELIVERY_DATA_FOR_AXIS_FORMAT = {
         "Blio" : None,
         "Acoustik" : None,
+        "AxisNow": None,
```

Comment:
Axis 360 added a new format we can't use. Putting it in here makes our Axis API client not worry about it.